.github/workflows/main.yml

```diff
@@ -0,0 +1,49 @@
+name: Publish docs via GitHub Pages
+on:
+  push:
+    branches:
+      - extensions-docs
```

Comment:
Should this be `master`?

Reply:
yes, correct, just updated the pull request!